app/src/main/java/org/mozilla/fenix/library/history/HistoryView.kt

```diff
@@ -15,70 +15,81 @@ import androidx.appcompat.widget.Toolbar
 import androidx.constraintlayout.widget.ConstraintLayout
 import androidx.core.content.ContextCompat
 import androidx.recyclerview.widget.LinearLayoutManager
-import io.reactivex.Observable
-import io.reactivex.Observer
-import io.reactivex.functions.Consumer
+import kotlinx.android.extensions.LayoutContainer
 import kotlinx.android.synthetic.main.component_history.*
 import kotlinx.android.synthetic.main.component_history.view.*
 import kotlinx.android.synthetic.main.delete_history_button.*
 import mozilla.components.support.base.feature.BackHandler
 import org.mozilla.fenix.R
 import org.mozilla.fenix.ext.asActivity
 import org.mozilla.fenix.ext.getColorIntFromAttr
-import org.mozilla.fenix.mvi.UIView
 
-class HistoryUIView(
-    container: ViewGroup,
-    actionEmitter: Observer<HistoryAction>,
-    changesObservable: Observable<HistoryChange>
-) :
-    UIView<HistoryState, HistoryAction, HistoryChange>(container, actionEmitter, changesObservable),
-    BackHandler {
+/**
+ * Interface for the HistoryViewInteractor. This interface is implemented by objects that want
+ * to respond to user interaction on the HistoryView
+ */
+interface HistoryViewInteractor {
+    fun onHistoryItemOpened(item: HistoryItem)
```

Comment:
Let's add some KDocs to these interface methods :). 📝

Reply:
I added the KDocs to the `HistoryInteractor` class, should they be here instead?

Reply:
Ah yes, since `HistoryViewInteractor` the KDocs should be there. They will be inherited by `HistoryInteractor`

Reply:
Done! :)